securedrop/models.py

```diff
@@ -30,7 +30,7 @@
     # statements has to be marked as noqa.
     # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
     from typing import Callable, Optional, Union, Dict, List, Any  # noqa: F401
-    from tempfile import _TemporaryFileWrapper  # noqa: F401
+    from tempfile import _TemporaryFileWrapper  # type: ignore # noqa: F401
```

Comment:
seems like we don't actually need this line for type checking? (I don't see `_TemporaryFileWrapper` in the type hints anywhere)

Reply:
```

Reply:
sorry by this line, I meant the import itself

Reply:
Yep, you're right.